### PR TITLE
Support testing targets in different workspaces.

### PR DESCRIPTION
**Pull request by tbosch**
@alexeagle How do test this?

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

**Comment by alexeagle** [comment]
I think the subtraction pattern here is why we didn't have test coverage for this:
https://github.com/bazelbuild/rules_nodejs/blob/master/.circleci/config.yml#L18
this was to work around issues with nested WORKSPACEs but I think we just need to correctly understand how they work.
(note that the Windows PR #36 is planning to drop the nested WORKSPACE as we also ran into issues there, hopefully learning how they are meant to work means we can remove that part of the PR)

File: internal/jasmine_node_test.bzl [hunk 1]
```diff
@@ -9,7 +9,8 @@ def jasmine_node_test(name, srcs, data = [], args = [], deps = [], **kwargs):
   )
 
   args = ["/".join([p
-      for p in [".", PACKAGE_NAME, "%s_devmode_srcs.MF" % name]
+      for p in [".", "external" if REPOSITORY_NAME else "", REPOSITORY_NAME[1:],
```
Comment (alexeagle) [inline internal/jasmine_node_test.bzl]: better to show what character is being stripped, either add a comment, or maybe
`REPOSITORY_NAME[len("."):]` to show which prefix we expect to be removed
or both
(isn't this a bug though, don't you want to split on `/` and take the second part?)

Comment (tbosch) [reply internal/jasmine_node_test.bzl]: Will change to use `len("@")`.
`REPOSITORY_NAME` is something like `@angular`.

**Comment by tbosch** [comment]
@alexeagle PTAL

**Pull request merged**